spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -2806,6 +2806,7 @@ def role_allows?(_)
                      :type      => :button,
                      :img       => "download_pdf.png",
                      :imgdis    => "download_pdf.png",
+                     :img_url   => "/assets/toolbars/download_pdf.png",
```

Comment:
Looks suprising, probably won't pass after the aforementioned change.. ... the generated url will be something with hexadecimals after `download_pdf`